### DISCUSSION TITLE
Is there a way to batch add/remove tags to many results?

### DISCUSSION THREAD

**Discussion by Andrew15-5** [monkeytypegame/monkeytype#5342] [Questions]
I run a lot of tests, but now I want to add a specific tag to many of them. How can I do this, if at all?

**Comment by Silverdev2482** [comment]
This would be really nice, because I'm switching layouts, and I want to tag all previous tests as my old layout.

**Reply by Andrew15-5** [reply]
For this exact use case, I had to make a script that would make a request to the server with an updated tag for a test. It's not hard, but I think there was a limit of about 100 requests, after which I had to fetch a new cookie token. So I had to re-log in about 7 times.
